src/sentry/static/sentry/app/views/performance/vitalDetail/vitalPercents.tsx

```diff
@@ -1,44 +1,67 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
+import Tooltip from 'app/components/tooltip';
 import {IconCheckmark, IconFire, IconWarning} from 'app/icons';
-import {t} from 'app/locale';
+import {t, tct} from 'app/locale';
 import space from 'app/styles/space';
+import {WebVital} from 'app/utils/discover/fields';
 import {formatPercentage} from 'app/utils/formatters';
 import theme, {Color} from 'app/utils/theme';
```

Comment:
We should not import `theme` like this, instead it is passed in through `props` when you are creating a styled component.

Reply:
Yes, I ran into this issue with https://github.com/getsentry/sentry/pull/23893. I think it's fine for now though; if we want to change that I think it should be in a separate PR.